tests/ui/const-generics/issues/issue-113462.rs

```diff
@@ -0,0 +1,11 @@
+struct S2<'b>(&'b ());
+
+#[derive(Default)]
+struct S<'a, const N: S2> {}
+//~^ ERROR missing lifetime specifier [E0106]
+//~| ERROR missing lifetime specifier [E0106]
+//~| ERROR parameter `'a` is never used [E0392]
+//~| ERROR `S2<'_>` is forbidden as the type of a const generic parameter
+//~| ERROR `S2<'_>` is forbidden as the type of a const generic parameter
+
```

Comment:
It is a bit strange that duplicate errors are reported, but when I run `﻿rustc +stage1`, the duplicates no longer appear.

Reply:
It caused by derive macro